source/ItemInfoDisplay.h

```diff
@@ -75,6 +81,10 @@ class ItemInfoDisplay {
 	mutable int hoverCount = 0;
 	bool hasHover = false;
 	mutable WrappedText hoverText;
+
+
+private:
+	const PlayerInfo *player = nullptr;
```

Comment:
isnt it a nulltptr by default?

Reply:
Yeah, but I've come to hate letting things be default, due to it causing multiple issues, so I prefer to make it explicit

Reply:
ok with me